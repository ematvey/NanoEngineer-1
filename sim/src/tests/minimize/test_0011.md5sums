stdout 133377BC72B22D95AD3271853E5E1FA4
stderr D41D8CD98F00B204E9800998ECF8427E
test_0011.xyz 8A463CBC86C9B6908D72B5035210082E
