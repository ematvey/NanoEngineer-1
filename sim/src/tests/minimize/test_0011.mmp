mmpformat 050130 required; 050217 optional
kelvin 300
group (Data)
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (0.457728, -0.139399, 0.871848, -0.104570) (5.314410) (0.000000, 0.000000, 0.000000) (1.000000)
datum (XY) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 0.000000, 1.000000) (0.000000, 1.000000, 0.000000)
datum (YZ) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (1.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000)
datum (ZX) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000) (0.000000, 1.000000, 0.000000)
egroup (Data)
group (Untitled)
mol (Chunk-C1) def
atom 1 (6) (-2742, -1034, -88) def
atom 2 (0) (-2528, -536, 457) def
bond1 1
atom 3 (0) (-3420, -1347, 100) def
bond1 1
atom 4 (6) (-2873, -297, -1434) def
bond1 1
atom 5 (0) (-2764, 456, -1325) def
bond1 4
atom 6 (0) (-3576, -416, -1724) def
bond1 4
atom 7 (6) (-1682, -2145, -213) def
bond1 1
atom 8 (0) (-1129, -2033, 311) def
bond1 7
atom 9 (0) (-2013, -2830, -95) def
bond1 7
atom 10 (6) (-1815, -831, -2417) def
bond1 4
atom 11 (0) (-1327, -265, -2601) def
bond1 10
atom 12 (0) (-2163, -1087, -3055) def
bond1 10
atom 13 (6) (-1051, -2048, -1691) def
bond1 7 10
atom 14 (0) (-1167, -2712, -2063) def
bond1 13
atom 15 (0) (-295, -1911, -1639) def
bond1 13
egroup (Untitled)
end1
group (Clipboard)
egroup (Clipboard)
end molecular machine part Untitled
