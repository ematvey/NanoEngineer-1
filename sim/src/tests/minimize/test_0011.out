======= test_0011.test =======
======= exitvalue =======
0
======= stderr =======
cutover to fine tolerance at 2
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_0011.mmp  > test_0011.xyz
final RMS gradient=3088.163537 after 5 iterations
======= test_0011.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Date and Time: Mon Nov 21 18:46:01 2005
# Input File:test_0011.mmp
# Output File: test_0011.xyz
# Trace File: test_0011.trc
# Number of Atoms: 15
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
#
# 0 columns:
#
0 3141.31 3508.14
1 3091.14 3465.47
2 3088.33 3463.03
3 3088.17 3462.89
4 3088.16 3462.88
5 3088.16 3462.88
# Done: Minimization final rms: 3088.163537, highForce: 3462.884504
======= test_0011.xyz =======
15
RMS=3088.163537
C -2.742203 -1.033957 -0.087676
X -2.527794 -0.535690 0.457268
X -3.420376 -1.347225 0.100024
C -2.873257 -0.296722 -1.434100
X -2.763845 0.456423 -1.324901
X -3.576379 -0.416166 -1.724167
C -1.681713 -2.145181 -0.213062
X -1.128670 -2.032848 0.311219
X -2.013207 -2.830379 -0.095122
C -1.814774 -0.831214 -2.417241
X -1.326625 -0.264749 -2.600952
X -2.163223 -1.087233 -3.055270
C -1.051050 -2.048037 -1.690823
X -1.167229 -2.712274 -2.063241
X -0.294656 -1.910749 -1.638958
